docs/advanced/compiler-annotations.md

```diff
@@ -851,3 +748,151 @@ print(tbl.get(foo)); // print(tbl[foo])
 tbl.set(1, "baz"); // tbl[1] = "baz"
 print(tbl.length()); // print(#tbl)
 ```
+
+### @vararg
```

Comment:
Could you add a `<DeprecatedInVersion deprecated="0.39.0" removed="TBD" />` like the other ones have?